okd/scripts/provision.sh

```diff
@@ -94,7 +102,7 @@ mkdir -p /root/install
 cp /install-config.yaml /root/install/
 
 # inject pull secret into install config
-cat /etc/installer/token >> /root/install/install-config.yaml
+echo "pullSecret: '$(cat /etc/installer/token)'" >> /root/install/install-config.yaml
```

Comment:
hm why do you change it? it will also expose the token under the provision console

Reply:
The first time I went through this, It wasn't obvious to me that the pullSecret file needed to be yaml formatted. I just wanted to past the secret to a file and have it work. 

i don't mind reverting this. it doesn't matter much to me.